packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-julia.vue

```diff
@@ -290,6 +301,12 @@ const disableRunButton = computed(
 		!datasetId.value
 );
 
+const updateStateExtras = () => {
```

Comment:
What does `Extra` means here?

Reply:
`Extra` holds the simulation  parameters, e.g.

```
"extra":{"numSamples":50,"totalPopulation":1000,"numIterations":10}
```

I dunno why it is named like that.

Reply:
it's named like that because that's what the field is called in the calibrate api request body